### ISSUE TITLE
[Bug]: Multiplied negative values increases amount

### ISSUE THREAD

**Issue by milankyncl** [dinerojs/dinero.js#713]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current behavior

After multiplying negative numbers it adds an extra amount into the Dinero object. 

```
const d = dinero({ amount: -200, currency: USD });    // { amount: 200 }  -> OK!
const m = multiply(d, 1);                             // { amount: -201 } -> not OK!
```

Happy to share further details.

Reproducible in the attached Code Sandbox.

### Expected behavior

To be the same, but just in a negative value.

### Steps to reproduce

https://codesandbox.io/s/fervent-violet-qkozxe?file=/main.js

### Version

2.0.0-alpha.13

### Environment

Node.js v14.18.0

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by johnhooks** [comment]
@milankyncl I think I why this bug might be happening. The core function [`transformScale`](https://github.com/dinerojs/dinero.js/blob/621f6c8d99dfbb41affd1bf78581d62c3ce12194/packages/core/src/api/transformScale.ts)  preforms a rounding operation in [`multiply`](https://github.com/dinerojs/dinero.js/blob/621f6c8d99dfbb41affd1bf78581d62c3ce12194/packages/core/src/api/multiply.ts). The logic was recently reworked in [`down`](https://github.com/dinerojs/dinero.js/blob/621f6c8d99dfbb41affd1bf78581d62c3ce12194/packages/core/src/divide/down.ts). I'll add some tests and dive into it soon.

**Comment by johnhooks** [comment]
@milankyncl Looks like this issue has already been fixed in PR #711.

**Comment by kdurek** [comment]
Hey, sadly same happens multiplying by -1
